Vagrantfile

```diff
@@ -102,7 +102,7 @@ Vagrant.configure("2") do |config|
     if ENV['OSQUERY_BUILD_CPUS']
       v.cpus = ENV['OSQUERY_BUILD_CPUS'].to_i
     else
-      v.cpus = 2
+      v.cpus = 4
```

Comment:
Nothing is wrong with that, but I felt that 2 was a bit low as a default for people who don't read the Vagrant file.